gabbi/runner.py

```diff
@@ -89,26 +96,48 @@ def run():
              'form package.module or package.module:class.'
     )
 
-    args = parser.parse_args()
+    args = parser.parse_args(argv[1:])
     host, port, prefix, force_ssl = utils.host_info_from_target(
         args.target, args.prefix)
 
-    # Initialize response handlers.
-    handler_objects = initialize_handlers(args.response_handlers)
+    response_handlers = initialize_handlers(args.response_handlers)
 
-    data = utils.load_yaml(handle=sys.stdin)
+    failfast = args.failfast
+    failure = False
+    for input_file in input_files:
+        params = (response_handlers, host, port, prefix, force_ssl, failfast)
```

Comment:
still no super keen on this param globbing, when execute is called you have to look elsewhere to see what it is called with, that seems bad to me.